src/bindings/python/tests/test_transformations/test_offline_api.py

```diff
@@ -74,7 +75,9 @@ def test_serialize_pass():
 
     func = get_test_function()
 
-    serialize(func, xml_path, bin_path)
+    m = Manager()
+    m.register_pass(Serialize(xml_path, bin_path))
```

Comment:
shouldn't this test be a part of `test_public_transformations.py`?

Reply:
Same for me. I think it can be a different test, as serialize function from openvino.runtime provides "a shortcut" for users.

Reply:
Moved.